src/pages/Notebook.pug

```diff
@@ -106,183 +107,216 @@ block article
+        This week we worked only on the immobilization and first analyzed a TdT reaction with conjugated primer in solution.
+        We decided to keep the stock concentration of the protein constant.
+        Therefore the resulting concentration of the primer in the TdT tailing reaction varied. 
+        The next day we began to purify the protein-primer conjugate by using nickel beads to analyze the concentrations and show whether the conjugates would be visible on the protein gel.
+        Later we analyzed both the purified samples and the flow-through on an agarose gel to see the effect on the purification and analyze the success of washing and purification.
+        Unfortunately, we estimated the primer and protein concentration wrong, so the references were not visible. 
+        Thus we repeated the experiment and re-analyzed the TdT samples in higher concentrations.
+        After the successful experiment, we carried out our first immobilization with the maleimide conjugates on a stick.
+        We assessed different reaction conditions, wash steps, and negative controls to gather as much information as possible from of the samples.
 
         # Week 20 - Testing of new immobilization conditions
         
-        Continuing our first experiments, we focused on analyzing whether a higher primer concentration influenced the running characteristics of the agarose gel. Therefore we created TdT reactions with primer concentrations from 20 to 1000 nM for the AT-rich as well as the GC-rich primer and analyzed them with an agarose gel.
+        Continuing our first experiments, we focused on analyzing whether a higher primer concentration influenced the running characteristics of the agarose gel.
+        Therefore we created TdT reactions with primer concentrations from 20 to 1000 nM for the AT-rich as well as the GC-rich primer and analyzed them with an agarose gel.
         
-        New TdT reactions were performed with the Thermofisher TdT and NEB buffer to check if the immobilizations worked better.
-        To increase the concentration of the primer-protein conjugate, we first started a TdT reaction and immobilized the primer during the reaction
-        Subsequently, repeated the experiment from last week with LCI-GC 1 on a smaller scale and tested the influence of the buffer on the adsorption of the anchor peptide.
+        New TdT reactions were performed with the ThermoFisher TdT and NEB buffer to check if the immobilization works better.
+        To increase the concentration of the primer-protein conjugate, we first started a TdT reaction and immobilized the primer during the reaction.
+        Subsequently, we repeated the experiment from last week with LCI-GC 1 on a smaller scale and tested the influence of the buffer on the adsorption of the anchor peptide.
```

Comment:
```suggestion
        Subsequently, we repeated the experiment from last week with conjugates (LCI-GC) on a smaller scale and tested the influence of the buffer on the adsorption of the anchor peptide.
```

Reply:
zu deinem kommentar @Tobias-J, LCI wird tatsächlich in Woche 11 als Ankerpeptid schon erwähnt. ist hier aber unklar gewesen, mit LCI-GC wir halt ein Konjugat aus Peptid LCI und GC-rich primer beschrieben... denke, man versteht das jetzt so